.github/workflows/lint.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
           VALIDATE_ALL_CODEBASE: false
           DEFAULT_BRANCH: main
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VALIDATE_PYTHON: true
```

Comment:
**[Question, non-blocking]** According to the [environment variable docs](https://github.com/super-linter/super-linter#environment-variables), `VALIDATE_PYTHON` should be set to `true` by default; are you experiencing unexpected behavior with the default config that this change is intended to resolve?

Reply:
No, you're completely right. We could eliminate it, the only reason I had it in here was an attempt to be about explicit which languages we're looking to lint, but it's really rather moot now that we're probably going to have more in this repo than we used to.